library-backend/docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3.5'
+services:
+  db:
+    container_name: db
+    image: "store/oracle/database-enterprise:12.2.0.1-slim"
+    restart: "unless-stopped"
+    ports:
+      - "1521:1521"
+    environment:
+      - url=jdbc:oracle:thin:@localhost:1521:ORCLCDB
```

Comment:
Why do you set these properties for the DB container?

The DB container doesn't need any URL to connect to.